utils/syntax_highlighting.tsx

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import hlJS from 'highlight.js/lib/highlight.js';
-import hljsActionscript from 'highlight.js/lib/languages/actionscript.js';
+import hlJS from 'highlight.js';
+import hljsActionscript from 'highlight.js/lib/languages/actionscript.js'
```

Comment:
Missing a semicolon at the end of this line.

Reply:
Done